lib/block_view.go

```diff
@@ -6881,6 +6974,15 @@ func (bav *UtxoView) _connectUpdateNFT(
 			txMeta.NFTPostHash.String())
 	}
 
+	// We only need to make these checks if we are past the Buy Now NFT Block Height
+	if postEntry.HasUnlockable && isBuyNow {
+		return 0, 0, nil, errors.Wrapf(RuleErrorCannotHaveUnlockableAndBuyNowNFT, "_connectUpdateNFT: ")
+	}
+
+	if isBuyNow && txMeta.MinBidAmountNanos == 0 {
```

Comment:
Again, unless there's a compelling reason to enforce this check in consensus, I would let the frontends do this validation.

Reply:
0 means cancel my bid.